lib/pin_input_text_field.dart

```diff
@@ -21,11 +21,24 @@ abstract class PinDecoration {
 
   final ObscureStyle obscureStyle;
 
+  final String errorText;
```

Comment:
Please add a comment for these variable.

Reply:
ok, will do that today.